src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/Sentence.kt

```diff
@@ -5,8 +5,17 @@ import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Sentence as Integ
 
 data class Sentence(
   val sentenceDate: LocalDate?,
+  val sentenceStatus: String?,
 ) {
   fun toSentence(): IntegrationApiSentence = IntegrationApiSentence(
     dateOfSentencing = this.sentenceDate,
+    isActive = toIsActive(this.sentenceStatus),
   )
 }
+private fun toIsActive(sentenceStatus: String?): Boolean? {
```

Comment:
This function name hurts my OCD. The question "isActive?", returning a boolean of true of false in certain cases of sentenceStatus, is already sufficient for this as a function name, no?
toIsActive implies we are changing the state of something, but we aren't doing this in my head?